inference-engine/samples/CMakeLists.txt

```diff
@@ -248,12 +248,11 @@ macro(ie_add_sample)
     endif()
     add_dependencies(ie_samples ${IE_SAMPLE_NAME})
 
-    if(COMMAND add_cpplint_target AND NOT IE_SAMPLE_EXCLUDE_CPPLINT)
-        if(c_sample)
-            set(custom_filters "-readability/casting")
+    if(COMMAND add_clang_format_target AND NOT IE_SAMPLE_EXCLUDE_CLANG_FORMAT)
+        add_clang_format_target(${IE_SAMPLE_NAME}_clang FOR_SOURCES ${IE_SAMPLE_SOURCES})
+        if(IE_SAMPLE_HEADERS)
+            add_clang_format_target(${IE_SAMPLE_NAME}_headers_clang FOR_SOURCES ${IE_SAMPLE_HEADERS})
         endif()
```

Comment:
can we merge these 2 `add_clang_format_target` into a single one? We can check both headers and source files using a single command.

Reply:
I tried but I didn't find to do it.